app/src/main/java/com/bottlerocketstudios/brarchitecture/ui/splash/SplashViewModel.kt

```diff
@@ -15,6 +15,10 @@ class SplashViewModel : BaseViewModel() {
     val unAuthEvent: SharedFlow<Unit> = MutableSharedFlow()
 
     init {
+        authenticate()
+    }
+
+    fun authenticate() {
```

Comment:
@MMurrayAtBR I see that you've created a public function just to test out this behavior. I'm currently exploring alternatives to test this with the code staying inside the `init` block. In the meantime, go ahead and add the `@VisibleForTesting` annotation to the `authenticate` function

Reply:
No longer applicable, so I'll resolve this conversation